frontend/src/metabase/admin/tasks/containers/Logs.jsx

```diff
@@ -80,23 +94,73 @@ export default class Logs extends Component {
   }
 
   render() {
-    const { logs } = this.state;
-    return (
-      <LoadingAndErrorWrapper loading={!logs || logs.length === 0}>
-        {() => (
-          <div
-            className="rounded bordered bg-light"
-            style={{
-              fontFamily: '"Lucida Console", Monaco, monospace',
-              fontSize: "14px",
-              whiteSpace: "pre-line",
-              padding: "1em",
-            }}
+    const { logs, selectedProcessUUID } = this.state;
+    const filteredLogs = logs.filter(
+      ev =>
+        !selectedProcessUUID ||
+        selectedProcessUUID === "ALL" ||
+        ev.process_uuid === selectedProcessUUID,
+    );
+    const processUUIDs = _.uniq(
+      logs.map(ev => ev.process_uuid).filter(Boolean),
+    ).sort();
+    const renderedLogs = filteredLogs.map(ev => {
+      const timestamp = moment(ev.timestamp).format();
+      const uuid = ev.process_uuid || "---";
+      return `[${uuid}] ${timestamp} ${ev.level} ${ev.fqns} ${ev.msg}`;
+    });
+
+    let processUUIDSelect = null;
+    if (processUUIDs.length > 1) {
+      const options = [
```

Comment:
You can just do something like this if you have a mix of static and dynamic options: 

```
<Select ...>
  <Option value="ALL" key="ALL">{t`All Metabase processes`}</Option>
  { processUUIDs.map(uuid => (
    <Option key={uuid} value={uuid}>
      <code>{uuid}</code>
    </Option>
  ))}
</Select>

Reply:
:heavy_check_mark: 

There was a bug in `Select` that prevented that. But I fixed it now :)